contracts/src/chainlink/cairo/emergency/SequencerUptimeFeed/library.cairo

```diff
@@ -62,7 +62,7 @@ func require_valid_round_id{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, ra
     with_attr error_message("invalid round_id"):
         assert_not_zero(round_id)
         # TODO: do we need to check if uint80 is overflown?
```

Comment:
@cds95 any ideas what's this about?

We should consider removing this TODO from here.

Reply:
@qperrot let's remove this comment as it doesn't make sense in the current context